sdk/batch/azure-batch/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 15.0.0b1 (2024-09-01)
+
+- Version (15.0.0b1) is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure Batch. For more information about this, and preview releases of other Azure SDK libraries, please visit https://azure.github.io/azure-sdk/releases/latest/python.html.
+
+### Breaking Changes
+
+- Remove certificates
+- Remove render licenses
+- Remove `CloudServiceConfiguration` for pool models and operations. `VirtualMachineConfiguration` is supported for pool configurations moving forward.
+
```

Comment:
are there any samples that show usage of the new library and how folks using track 1 would move to this version?

Reply:
We have samples that is attached to our api view that shows usage of the new library that we can move to a `samples` directory here. 

We plan on making a migration guide before we GA. This is just the beta release.